packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -112,6 +112,9 @@ class EmbeddedContentConsumer extends React.Component {
     // Mount the provided source as the application into the content wrapper.
     this.xfcFrame = Consumer.mount(this.embeddedContentWrapper, this.props.src, frameOptions);
 
+    // Set additional style on xfcFrame
+    this.xfcFrame.iframe.classList.add(cx('xfc-iframe-margin'));
```

Comment:
Do you need a separate class just for the margin?  Or is this the class that you want for your iFrame?  You normally don't name your class based on styles.

Reply:
This is to set the `margin-top` and `margin-bottom` for the default case (before focus) so that when we apply the style during focus, the content doesn't move. I think we need it. I'll try to rename it.